caput/containers/tod.py

```diff
@@ -50,53 +70,55 @@ def time(self):
         return time
 
     @classmethod
-    def from_mult_files(
+    def from_mult_files(  # noqa: D417
         cls,
-        files,
-        data_group=None,
-        start=None,
-        stop=None,
-        datasets=None,
-        dataset_filter=None,
-        **kwargs,
-    ):
-        """Create new data object by concatenating a series of objects.
-
-        Accepts any parameter for :func:`concatenate` (which controls the
+        files: FileLike | Sequence[FileLike],
+        data_group: GroupLike | None = None,
+        start: int | dict | None = None,
+        stop: int | dict | None = None,
+        datasets: Sequence[str] | None = None,
+        dataset_filter: Callable | None = None,
+        **kwargs: dict,
+    ) -> TOData:
+        r"""Create new data object by concatenating a series of objects.
+
+        Accepts any parameter for :py:func:`concatenate` (which controls the
```

Comment:
```suggestion
        Accepts any parameter supported by :py:func:`concatenate` (which controls the
```
Also: the ":py:func:`concatenate`" isn't being turned into a link for some reason.